packages/minter-contracts/ligo/fa2_modules/fa2_allowlist/allowlist_base.mligo

```diff
@@ -0,0 +1,23 @@
+(* Base allowlisting module.
+ * Import it in a contract if it is planned to be extended with allowlisting
+ * extensions.
```

Comment:
I think this should better explain that only using this module won't actually add any allowlisting (on its own).